app/_src/kubernetes-ingress-controller/guides/migrating-from-ingress-to-gateway.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Migrating from Ingress to Gateway
+type: how-to
+purpose: |
+    How to migrate from Ingress API to Gateway API
+---
+
+## Prerequisites
+
+This guide will use git and [go][go] to download and execute the [`ingress2gateway`][ingress2gateway]
+tool.
+
+```bash
+git clone https://github.com/kong/ingress2gateway
+cd ingress2gateway
+git checkout preview
+make build
+export PATH=$PATH:$(pwd)
+```
+
+[go]: https://go.dev/dl/
+[ingress2gateway]: https://github.com/kubernetes-sigs/ingress2gateway
+
+## Convert all your yaml files
+
+In order to migrate your resources from ingress to gateway, you need all your ingress-based
```

Comment:
```suggestion
In order to migrate your resources from `Ingress` API to Gateway API you need all your `Ingress`-based
```

Not sure how we'd like to approach this. Whatever we choose let's be consistent.